recipes.adoc

```diff
@@ -66,3 +66,20 @@ custom distributions.
 | Profile | UEFI | ACPI | DT | SBI | SMBIOS
 | >= RVA20 | EBBR cite:[EBBR] | optional, >= 6.6 | optional, >= v0.3 | >= 2.0 | optional, >= 3.6.0
 |===
+
+=== BRS-M Recipe
+
+BRS-M is used for cases that do not require UEFI. BRS-M is the
+smallest recipe. This recipe does not require UEFI boot and runtime
+service implementations. In some boot scenarios, non-UEFI firmware
+can also support boot, they only focus on the boot process and support
+the necessary interaction interfaces.
+<<recipe-brs-m-guidance, See additional guidance>>.
```

Comment:
In this merge request I only find what BRS-M does not offer but not any definition of what it should provide. This is insufficient to define a recipe.

Reply:
What do you suggest should be added, where?

Reply:
The reason it does not define these things is that they are provided elsewhere / not needed. You can make cake without flour, and people still call it a cake. You can make bread without yeast, and it is flatbread. You can make Linux without block device support, and it is still a kernel. Similarly, BRS-M is a reduced model, but it is still a recipe.